### PR TITLE
fix: handle separately 401 and 403 api errors in UI

### PR DESCRIPTION
**Pull request by frascuchon**
Since the changes included in #1713, users without permission trying to delete datasets from UI will hit into a re-login operation.

This PR handles separate authentication errors (401) and forbidden errors (403).

We should find a better way to handle error codes and info in general.

Refs: #1713

### CODE REVIEW

File: frontend/plugins/vuex-orm-axios.js [hunk 1]
```diff
@@ -70,13 +70,13 @@ export default ({ $axios, app }) => {
         break;
       case 404:
         Notification.dispatch("notify", {
-          message: "Warning: " + error.response.data.detail,
+          message: "Warning: " + messageDetail.params.detail,
```
Comment (keithCuniah) [inline frontend/plugins/vuex-orm-axios.js]: Some code smeller recommends to concatenate strings with backtick like so : (backtick)Warning : ${messageDetail.params.detail}(backtick)

File: frontend/plugins/vuex-orm-axios.js [hunk 2]
```diff
@@ -70,13 +70,13 @@ export default ({ $axios, app }) => {
...
           type: "warning",
         });
         break;
       default:
         Notification.dispatch("notify", {
-          message: error,
+          message: "Error: " + messageDetail.params.detail,
```
Comment (keithCuniah) [inline frontend/plugins/vuex-orm-axios.js]: Some code smeller recommends to concatenate strings with backtick like so : (backtick)Error: ${messageDetail.params.detail}(backtick)

Comment (frascuchon) [inline frontend/plugins/vuex-orm-axios.js]: We could apply your suggestion here, isn't it?
```suggestion
          message: `Error:  ${messageDetail.params.detail}`,
```

File: frontend/plugins/vuex-orm-axios.js (hunk 1 above)
Comment (frascuchon) [inline frontend/plugins/vuex-orm-axios.js]: like that ? 

```suggestion
          message: `Warning: ${messageDetail.params.detail}`,
```

Comment (keithCuniah) [reply frontend/plugins/vuex-orm-axios.js]: yes like your correction

File: frontend/plugins/vuex-orm-axios.js (hunk 2 above)
Comment (keithCuniah) [reply frontend/plugins/vuex-orm-axios.js]: yes, but don't forget the brackets  around the variable : `Error:  ${messageDetail.params.detail}`,

**Pull request merged**